src/app/helpers/helpers.component.ts

```diff
@@ -231,6 +231,16 @@ export class HelpersComponent implements OnInit, OnDestroy {
         column: helpers.distCenterAdmin, width: '160'
       });
     }
+    if (this.context.isAllowed(Roles.lab)) {
+      r.push({
+        column:helpers.lab,width:'120'
+      });
+      r.push({
```

Comment:
לא הבנתי למה

Reply:
אתה רוצה מצב שבו LAB אחד יתן הרשאה לLAB אחר

Reply:
It will be visible to lab or admin. so both can assign "lab" to others.

Reply:
I see - did you confirm that other environments don't see this? From the code it seems to me that any admin will see  this flag

Reply:
verified